design/cli-install-changes.md

```diff
@@ -0,0 +1,156 @@
+      --get-snapshot-locations                          Display snapshot locations
+      --current-snapshot-locations                      displays the current default volume snapshot locations (NEW)
+      --use-snapshot-locations mapStringString          sets the list of unique volume providers and default volume snapshot location (provider1:location-01,provider2:location-02,...) (was `default-volume-snapshot-locations`)
+
+      --set-default-location                            configuration to create a default locations. Format is bucket=value,prefix=value,plugin-name=value,snapshot-location=true/false. Optional.
+
+      --provider string                                 provider name for backup and volume storage - (DEPRECATED)
+      --no-default-backup-location                      flag indicating if a default backup location should be created. Must be used as confirmation if --bucket or --provider are not provided. Optional. (DEPRECATED)
+      --use-volume-snapshots                            whether or not to create snapshot location automatically. Set to false if you do not plan to create volume snapshots via a storage provider. (default true) (DEPRECATED)
+```
+
+4) `velero plugin`
+Component specific configuration for plugins.
+
+```
+      --add stringArray                           Add plugin container images to install into the Velero Deployment
+      --list                                      Get information for all plugins on the velero server (was `get`)
+      --remove                                    Remove a plugin
+      --plugin-dir string                         directory containing Velero plugins (default "/plugins")
+```
+
+5) `velero restic`
+Component specific configuration for restic operations.
+
+```
+      --default-prune-frequency duration          how often 'restic prune' is run for restic repositories by default. Optional.
+      --pod-annotations mapStringString           annotations to add to the Velero and restic pods. Optional. Format is key1=value1,key2=value2
+      --pod-cpu-limit string                      CPU limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --pod-cpu-request string                    CPU request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --pod-mem-limit string                      memory limit for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --pod-mem-request string                    memory request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+      --create                                    create restic deployment. Optional. (was `use-restic`)
+      --repo                                      Work with restic repositories
+      --restic-timeout duration                   how long backups/restores of pod volumes should be allowed to run before timing out (default 1h0m0s)
+```
+
+#### Example
```

Comment:
reiterating previous comment, we need to think about what this looks like for a GitOps user.